oak_functions/loader/src/server.rs

```diff
@@ -777,24 +779,36 @@ mod tests {
     };
     use maplit::hashmap;
 
-    fn create_test_wasm_handler() -> WasmHandler {
-        let logger = Logger::for_test();
-        let lookup_data = Arc::new(LookupData::for_test(hashmap! {}));
-        let lookup_factory =
-            LookupFactory::new_boxed_extension_factory(lookup_data, logger.clone())
-                .expect("could not create LookupFactory");
+    #[test]
+    fn test_start_from_empty_endpoints() {
+        fn check_empty(endpoint: &mut Endpoint) {
+            let receiver = &mut endpoint.receiver;
+            assert_eq!(TryRecvError::Empty, receiver.try_recv().unwrap_err());
+        }
+        let (mut module, mut runtime) = channel_create();
+        check_empty(&mut module);
+        check_empty(&mut runtime);
+    }
 
-        let wasm_module_bytes = test_utils::create_some_wasm_module_bytes();
+    #[tokio::test]
+    async fn test_crossed_write_read() {
+        async fn check_crossed_write_read(endpoint1: &mut Endpoint, endpoint2: &mut Endpoint) {
+            let message = String::from("Message").into_bytes();
```

Comment:
What's the `from` for? Does `"Message".as_bytes()` work? Also messages are really binary, so maybe just make up a binary Vec with some numbers?

Reply:
Done.